3drepo/3drepo.io
# Issue 5267 - Ticket filters: Date

This fixes #5267

#### Description
Added support and validation for date filters

#### Test cases
Test normal date as well as ranges
- Make sure validation is correct;
- Check they are correctly rendered in the filter chip;
- Make sure in validation, the invalid dates are correctly disabled
